jfix-stdlib-concurrency/src/test/java/ru/fix/stdlib/concurrency/futures/PendingFutureLimiterTest.java

```diff
@@ -282,6 +283,26 @@ public void waitAll_releases_thread_after_last_future_is_completed() throws Exce
         assertTimeoutPreemptively(assertingTaskTimeout, () -> limiter.waitAll(largeTimeout));
     }
 
+    @Test
+    public void enqueueBlocking_blocks_invoking_thread_when_limit_is_reached() throws Exception {
+        PendingFutureLimiter limiter = new LimiterBuilder()
+                .executionTimeLimit(0)
+                .enqueueTasks(3)
+                .build();
+        AtomicBoolean ai = new AtomicBoolean(false);
```

Comment:
Please, give meaningful name, like enqueBlockingIsCompleteAndReturned instead of ai

Reply:
fixed